encoder.py

```diff
@@ -0,0 +1,38 @@
+import i2c
+import mux
+
+ENCODER_ADDR = 0x36
+ANG_REG = 0xE
+ANG_MASK = 0xFFF
+
+# Ang Range: 0 - FFF
+
+class Encoder():
+    def __init__(self, i2c):
+        self.i2c = i2c
+
+    def getAng():
```

Comment:
```suggestion
    def getAng(self):
```
same applies to the other methods below

Reply:
> same applies to the other methods below